openvdb_cmd/vdb_render/CMakeLists.txt

```diff
@@ -71,7 +71,11 @@ target_link_libraries(vdb_render
 
 if(WIN32)
   # @note OPENVDB_OPENEXR_STATICLIB is old functionality and should be removed
-  get_target_property(ILMBASE_LIB_TYPE IlmBase::Half TYPE)
+  if (TARGET Imath::Imath)
+    get_target_property(ILMBASE_LIB_TYPE Imath::Imath TYPE)
+  else()
+    get_target_property(ILMBASE_LIB_TYPE IlmBase::Half TYPE)
+  endif()
```

Comment:
@Idclip - any thoughts on this particular change? This is the only bit that doesn't touch the switch to 8 core builds.

Reply:
I don't know precisely, but I think it's most likely the version of imath/half being provided from vcpkg. It's only failing in the Windows builds.

Reply:
The change is consistent with what's needed for upgrading to OpenEXR 3.X when they did the Imath separation. And indeed it looks like vcpkg just updated to OpenEXR 3.1 a mere 12 days ago:
- https://github.com/microsoft/vcpkg/pull/26862

Reply:
Thanks for confirming!